tests/installation/language_keyboard/switch_keyboard_layout.pm

```diff
@@ -8,8 +8,14 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 use Test::Assert ':all';
+use testapi;
+use Utils::Backends;
 
 sub run {
+    if (!is_qemu) {
+        record_info('Skip', 'This module only run on installations not using remote protocol');
```

Comment:
s/This module only run on installations not using remote protocol/Switch keyboard not testeable for non-qemu scenarios/
(I would recommend you this above as I don't know what you are referring with remote protocol, I guess you mean remote workers).